torch/csrc/autograd/profiler.cpp

```diff
@@ -88,39 +95,59 @@ void popRange() {
   }
 }
 
-void enableProfiler(ProfilerState new_state) {
+void enableProfiler(ProfilerConfig config) {
+  ProfilerState new_state = config.state;
   AT_ASSERT(new_state != ProfilerState::Disabled);
   if (new_state == ProfilerState::NVTX && !cuda_stubs->enabled())
-    throw std::runtime_error("Can't use NVTX profiler - PyTorch was compiled without CUDA");
+    throw std::runtime_error(
```

Comment:
nit: this diff is sprinkled with formatting changes that are unnecessary. pytorch's column limit is 120 chars, so all of the 80-limit changes are unncessary

Reply:
@soumith , I agree that it makes things harder for review. This is an issue with tooling I see here.. I apply standard Facebook code formatting to a block of code I change. But it can touch other parts. I didn't reformat the whole file.

for me it is hard to spend time manually undoing changes which are being done automatically by our Atom integration. Do we have a good wiki page which describes how one can integrate pytorch code formatting into Atom?

Reply:
btw, the code looked really bad below. Like doing "for()" instead of "for (...)". This is not related to 120 characters limit.

Reply:
I thought we made sure the flake / linter in fbcode is now properly specified to respect some pytorch formatting. cc: @ezyang if you have any suggestions here.

Reply:
Well, my use-case is the following. I type code, it causes linter warning or just looks ugly. I select this code, type cmd + c and it formats the code. Sometimes it can touch more stuff for some reason. Btw, things that it touched are not due to 120 characters mainly but dew to missing whitespaces. Which are really annoying and make the code hard to read. 

IMO if this is just a few places which make code nicers - it is fine. If this is a really large file being fully rewritten - thats different.

Reply:
I've primarily looked at the Python lint situation, and I am not familiar with what is going on with the C++.

But there's a very simple fix for this problem; just do the reformatting as a commit beforehand, and then stack the actual change on top.